### PR TITLE
fix(card-in-card): add video variation

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#5458 

### Description

Add video variation for `<dds-card-in-card>`
Change rendering of video from cta-card copy to heading in react and web components. Not a breaking change, but could potentially effect users.

Screen Shot 2021-04-07 at 5 11 09 PM

### Changelog

**New**

- video knob in default story

**Changed**

- extending from cardCTA instead of featureCard
- updated styles
- video title to be rendered in heading instead of copy in both react and web components

**Removed**

- card-in-card-footer not necessary. using card-cta-footer

### CODE REVIEW

File: packages/web-components/src/components/card-in-card/__stories__/card-in-card.stories.ts [hunk 1]
```diff
@@ -35,13 +36,32 @@ export const Default = ({ parameters }) => {
       </dds-card-in-card-image>
       <dds-card-eyebrow>${eyebrow}</dds-card-eyebrow>
       <dds-card-heading>${heading}</dds-card-heading>
-      <dds-card-in-card-footer>
+      <dds-card-cta-footer>
         ${ArrowRight20({ slot: 'icon' })}
-      </dds-card-in-card-footer>
+      </dds-card-cta-footer>
```
Comment (jeffchew) [inline packages/web-components/src/components/card-in-card/__stories__/card-in-card.stories.ts:41]: If this is a minor breaking change we should note this in the release notes. Not sure if AEM in particular is already using this or not.

Comment (ariellalgilmore) [reply packages/web-components/src/components/card-in-card/__stories__/card-in-card.stories.ts:41]: oh good call! agreed!

**Comment by shixiedesign** [comment]
Hi Ari! Percy looked great. Thanks for catching that padding error. There's only 1 issue - I noticed the Video title is rendered as Card copy, instead of Card headline. This might be an existing issue with Video card. Lemme know if it is in scope. Here's James's issue on this topic #5538 
 
image

**Comment by shixiedesign** [comment]
Here are 2 other bugs that I believe should be in scope for this PR to fix: 
1. Hover on content card should also result in darkening the media. Currently there are 2 different results hovering on card and on media: 
Apr-07-2021 15-40-26

 2. Focus state should be showing blue outline **inside** the boundaries of the component 
 
image

**Comment by ariellalgilmore** [comment]
accidentally merged old commits. will create new clean PR

**Comment by ariellalgilmore** [comment]
Thanks @shixiedesign! everything has been updated! Is there a dev issue to reference for the updates to video info rendering in cta-card heading instead?

**Review by jeffchew — APPROVED** [review]
LGTM!

**Review by oliviaflory — APPROVED** [review]
Looks good! Confirmed the things Shixie caught are fixed 🎉

**Pull request merged**